include/picongpu/particles/ionization/byCollision/ThomasFermi/ThomasFermi_Impl.hpp

```diff
@@ -298,7 +293,7 @@ namespace ionization
 
                 /* Returns the new number of bound electrons for an integer number of macro electrons */
                 IonizationAlgorithm ionizeAlgo;
-                numNewFreeMacroElectrons = ionizeAlgo(
+                unsigned int numNewFreeMacroElectrons = ionizeAlgo(
```

Comment:
Did it because I found it like that [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/c3054c36ef5cc482233b4779fc9d8dbaed8d1967/src/picongpu/include/particles/creation/creation.kernel#L141) but I already assumed that it should be `uint32_t` there as well. ^^;